tools/topology/sof/pipe-low-latency-capture-sched.m4

```diff
@@ -0,0 +1,67 @@
+# Low Latency Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+#  host PCM_C <--B5-- volume(0C) <--B4-- source DAI0
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`pga.m4')
+include(`mixercontrol.m4')
+
+#
+# Controls
+#
+# Volume Mixer control with max value of 32
+C_CONTROLMIXER(PCM PCM_ID Capture Volume, PIPELINE_ID,
+	CONTROLMIXER_OPS(volsw, 256 binds the mixer control to volume get/put handlers, 256, 256),
+	CONTROLMIXER_MAX(, 40),
+	false,
+	CONTROLMIXER_TLV(TLV 32 steps from -64dB to 0dB for 2dB, vtlv_m64s2),
+	Channel register and shift for Front Left/Right,
+	LIST(`	', KCONTROL_CHANNEL(FL, 0, 0), KCONTROL_CHANNEL(FR, 0, 1)))
+
+#
+# Components and Buffers
+#
+
+# Host "Low Latency Capture" PCM
+# with 0 sink and 2 source periods
+W_PCM_CAPTURE(PCM_ID, Low Latency Capture, 0, 2)
+
+# "Capture Volume" has 2 sink and source periods for host and DAI ping-pong
+W_PGA(0, PIPELINE_FORMAT, 2, 2, LIST(`		', "PIPELINE_ID PCM PCM_ID Capture Volume"))
+
+# Capture Buffers
+W_BUFFER(0, COMP_BUFFER_SIZE(2,
+	COMP_SAMPLE_SIZE(PIPELINE_FORMAT), PIPELINE_CHANNELS, SCHEDULE_FRAMES),
+	PLATFORM_DAI_MEM_CAP)
+W_BUFFER(1, COMP_BUFFER_SIZE(2,
+	COMP_SAMPLE_SIZE(PIPELINE_FORMAT), PIPELINE_CHANNELS, SCHEDULE_FRAMES),
+	PLATFORM_HOST_MEM_CAP)
+
+#
+# Pipeline Graph
+#
+#  host PCM <--B1-- volume <--B0-- source DAI0
+
+P_GRAPH(pipe-ll-capture-PIPELINE_ID, PIPELINE_ID,
+	LIST(`		',
+	`dapm(N_PCMC(PCM_ID), N_BUFFER(1))',
+	`dapm(N_BUFFER(1), N_PGA(0))',
+	`dapm(N_PGA(0), N_BUFFER(0))'))
+
+#
+# Pipeline Source and Sinks
+#
+indir(`define', concat(`PIPELINE_SINK_', PIPELINE_ID), N_BUFFER(0))
+indir(`define', concat(`PIPELINE_PCM_', PIPELINE_ID), Low Latency Capture PCM_ID)
+
+W_PIPELINE(N_PGA(0), SCHEDULE_PERIOD, SCHEDULE_PRIORITY, SCHEDULE_FRAMES, SCHEDULE_CORE, SCHEDULE_TIME_DOMAIN, pipe_ll_schedule_plat)
```

Comment:
@juimonen the volume is the scheduling comp for this pipeline?

Reply:
@ranj063 this is quite direct copy from byt pipeline with mixer. There the second branch's scheduling component is SRC. I know that without the scheduling component definition this pipeline will not get "complete" and for e.g. all set/get fails. Here we don't have SRC so I put the nearest comp to demux as scheduler. So I'm not sure actually what should be the "scheduler" here.. there's only buffer->pga->buffer->pcm.

Reply:
@juimonen have you pulled the latest code for the mixer pipeline for BYT? Thats not the case anymore. Its been changed to take the scheduling comp as an argument now and it is actually the DAI thats the scheduling comp now.

Reply:
@ranjani ok will check it

Reply:
@ranj063 I have to build against cml-004-stable and I think the change is not there yet...

Reply:
@juimonen without that change you will not be able to propagate "copy" beyond the current pipeline. So you need have the same scheduling comp.

Reply:
@ranj063 little bit related, I see your change on driver for topology.h for sof_comp_type enum... I don't see that in FW side or am I missing something? I have to add the mux/demux to driver side...